frontend/fingertips-frontend/components/forms/IndicatorSearchForm/index.tsx

```diff
@@ -0,0 +1,58 @@
+import { ErrorText, FormGroup, SearchBox } from 'govuk-react';
+import { spacing } from '@govuk-react/lib';
+import styled from 'styled-components';
+import { IndicatorSearchFormState } from './indicatorSearchActions';
+
+const govukRed = '#d4351c';
+const govukErrorBorderWidth = '2px';
+
+const StyledSearchBox = styled(SearchBox)(
+  spacing.withWhiteSpace({ marginBottom: 6 })
+);
+
+export const IndicatorSearchForm = ({
+  searchFormState,
+}: {
+  searchFormState: IndicatorSearchFormState;
```

Comment:
I'm not sure about naming this `searchFormState` this clashes with the naming we have used for the form in the home page. 

I'd be happy to go with `indicatorSearchFormState` instead.